data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -16,23 +16,23 @@ if ! release=$( podman inspect {{.ReleaseImage}} -f '{{"{{"}} index .RepoDigests
 	release="{{.ReleaseImage}}"
 fi
 
-MACHINE_CONFIG_OPERATOR_IMAGE=$(podman run --quiet --rm ${release} image machine-config-operator)
-MACHINE_CONFIG_OSCONTENT=$(podman run --quiet --rm ${release} image machine-os-content)
-MACHINE_CONFIG_ETCD_IMAGE=$(podman run --quiet --rm ${release} image etcd)
-MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE=$(podman run --quiet --rm ${release} image kube-client-agent)
-MACHINE_CONFIG_INFRA_IMAGE=$(podman run --quiet --rm ${release} image pod)
+MACHINE_CONFIG_OPERATOR_IMAGE=$(podman run --quiet --rm --net=host ${release} image machine-config-operator)
```

Comment:
Nit; some of these, in fact most could be `--net=none` (not that it matters really but being able to disable networking is a powerful container feature in general and we should keep it in mind).

Also may be worth a prep commit which abstracts a `bootkube-podman-run` helper or so.